tools/site_and_regional/plumber2_usermods.ipynb

```diff
@@ -0,0 +1,344 @@
+{
```

Comment:
Awesome thanks for that @TeaganKing. 

Since we want to use conda for our python environments over pip, we should figure out how to do this with conda. It probably would be it's own environment.

The reason to use conda is that we already have support for it, it's the standard for NCAR HPC, and there are issues with using both pip and conda. A CISL talk I went to endorsed not using pip unless you really have to.

Reply:
I think this would be easy to do. If so I'll do it, but if not I'll create an issue.